src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -124,6 +124,26 @@ public class RepositoryOptions extends OptionsBase {
       help = "Scale all timeouts related to http downloads by the given factor")
   public double httpTimeoutScaling;
 
+  @Option(
+      name = "http_connector_attempts",
+      defaultValue = "0",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "The maximum number of attempts for http downloads. If set to 0, "
+              + "default number of 8 attempts is used.")
+  public int httpConnectorAttempts;
+
+  @Option(
+      name = "http_connector_retry_max_timeout",
+      defaultValue = "0",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "The maximum timeout for http download retries. If set to 0, "
+              + "a default value is used.")
+  public int httpConnectorRetryMaxTimeout;
```

Comment:
use the `Duration` type instead: see example https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/buildeventservice/BuildEventServiceOptions.java;drc=f63bb58023840454ba4997175487865b0ae8ec74;l=43

Reply:
Done